### PR TITLE
Added the new Utilities package and cleaned up duplicates

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
# Reality Collective - Reality Toolkit Pull Request

## Overview

Added the new Utilities package and cleaned up duplicates

### CODE REVIEW

File: Editor/ServiceProfileInspector.cs [hunk 1]
```diff
@@ -134,22 +130,30 @@ protected void RenderConfigurationOptions(bool forceExpanded = false)
...
+                {
+                    enterChildren = false;
+                }
+
+                if (!excludedProperties.Contains(iterator.name))
+                {
                     EditorGUILayout.PropertyField(iterator, true);
+                }
             }
-            if (CheckForCustomProperties())
```
Comment (FejZa) [inline Editor/ServiceProfileInspector.cs:146]: Why was this removed without replacement? This is a new feature Ronny added recently.

File: Runtime/Extensions/TypeExtensions.cs [hunk 2]
```diff
@@ -2,203 +2,24 @@
...
 using System.Collections.Generic;
 using System.Linq;
-using System.Reflection;
-using System.Runtime.InteropServices;
 using UnityEngine;
 
 namespace RealityToolkit.ServiceFramework.Extensions
 {
-    public static class TypeExtensions
+    public static class SFTypeExtensions
```
Comment (FejZa) [inline Runtime/Extensions/TypeExtensions.cs:13]: Why the rename?

**Review by FejZa — CHANGES_REQUESTED** [review]

**Comment by FejZa** [comment]
I thiknk this PR is not in sync with dev. It looks like some of the recent changes from @Cangi were removed again.

File: Runtime/Extensions/TypeExtensions.cs (hunk 2 above)
Comment (SimonDarksideJ) [reply Runtime/Extensions/TypeExtensions.cs:13]: Because it conflicts with the version in the Utilities project.  Not entirely sure why, but a rename was the only thing to clear it

File: Editor/ServiceProfileInspector.cs (hunk 1 above)
Comment (SimonDarksideJ) [reply Editor/ServiceProfileInspector.cs:146]: Lol, because it wasn't needed

**Review by FejZa — APPROVED** [review]
